### PR TITLE
Added Google Interactive API to interact with the Camera

### LINKED ISSUE #722: Google Now voice interaction with the application
**Actual Behaviour**

There is no voice interaction with the application. 

**Expected Behaviour**

Establishing Google Now voice interaction to take a photo, switch camera and more. 

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by Subhankar29**
Fix #722 

Changes: 

- Open camera using Google Now command. 

- Choose between front and rear camera

- Click picture on "cheese"

Steps to test the command:
-> Open Google Now or give command "OK Google".
-> Give the command "Take a selfie" or "Take a photo".
-> Choose the camera front or back by giving the command ->"Front" or "Selfie" for the front camera. "Rear" or "Back" or "Normal" for the rear camera. 
-> Give command "Cheese" or "yes" to take the photo.

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/opencamera/Camera/CameraActivity.java [hunk 1]
```diff
@@ -772,8 +774,91 @@ protected void onResume() {
...
+		rear.addSynonym("Normal");
+
+		VoiceInteractor.PickOptionRequest.Option front = new VoiceInteractor.PickOptionRequest.Option("Front Camera", 1);
+		front.addSynonym("Front");
+		front.addSynonym("Selfie");
+		front.addSynonym("Forward");
+
+		CameraActivity.this.getVoiceInteractor()
+				.submitRequest(new VoiceInteractor.PickOptionRequest(
+						new VoiceInteractor.Prompt("Which camera would you like to use?"),
```
Comment (the-dagger) [inline app/src/main/java/org/fossasia/phimpme/opencamera/Camera/CameraActivity.java]: Move these strings to strings.xml

File: app/src/main/java/org/fossasia/phimpme/opencamera/Camera/CameraActivity.java [hunk 2]
```diff
@@ -772,8 +774,91 @@ protected void onResume() {
...
+	private void rearCamera() {
+		int cameraId = getNextCameraId();
+		this.preview.setCamera(cameraId);
+		return;
+	}
+
+	private void asktakePicture() {
+		VoiceInteractor.PickOptionRequest.Option option = new VoiceInteractor.PickOptionRequest.Option("cheese", 2);
+		option.addSynonym("ready");
+		option.addSynonym("go");
```
Comment (the-dagger) [inline app/src/main/java/org/fossasia/phimpme/opencamera/Camera/CameraActivity.java]: Same, or make this a constant field.

File: app/src/main/java/org/fossasia/phimpme/opencamera/Camera/CameraActivity.java [hunk 3]
```diff
@@ -772,8 +774,91 @@ protected void onResume() {
...
+		option.addSynonym("take it");
+		option.addSynonym("ok");
+
+		CameraActivity.this.getVoiceInteractor()
```
Comment (the-dagger) [inline app/src/main/java/org/fossasia/phimpme/opencamera/Camera/CameraActivity.java]: This isn't an inner class, you can simply call `getVoiceInteractor()`

**Comment by Subhankar29** [comment]
@the-dagger Please review I have made the necessary changes.

**Comment by Subhankar29** [comment]
@pa1pal @mohitmanuja @anantprsd5 @vinaysajjanapu please review it. Require for further implementation.

**Comment by pa1pal** [comment]
@Subhankar29 Please add steps in the description to test the commands

**Comment by Subhankar29** [comment]
@pa1pal Sure, please refer it.

**Pull request merged**